packages/x-date-pickers/src/locales/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from './ruRU';
```

Comment:
Nitpick: usually we tend to add new options to the bottom of the file/list instead of the top 🤔

Reply:
it's implicit at first glance, why isn't it alphabetical?

Reply:
No particular reason.
We tend to nitpick the files we author more closely than those, which receive public contributions, but even then—import/export order is not enforced in any way.
That's why I prefixed this comment with a `nitpick`.
Maybe I did not make myself clear enough with that. 🙈 I meant that it does not matter, but just mentioned a general usage case. 😉